content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md

```diff
@@ -0,0 +1,67 @@
+
+FreeWear has greatly simplified the process of selling merchandise, having built their own online store, and handling the printing and shipping processes as well as the process of donating a portion of the sales straight to the FOSS project/organisation. Moreover, most of the world can receive merchandise from FreeWear. A partnership with FreeWear does not involve any upfront costs from the FOSS project, as a portion of the sales pays from expenses incurred such as printing.   
+
+Any FOSS organisations who are interested in working with FreeWear can contact them through email at admin(@)freewear.org   
+
+A list of the available merchandise, as well as the prices in Euros (EUR) and US Dollars (USD), follows, along with an image showing all the available products. Note that prices in US Dollars vary depending on exchange rates and are indicative only.   
+
+| Product Code & Name        | Price in EUR (€) | Indicative Price in USD ($) |
+|----------------------------|------------------|-----------------------------|
+| FW0468 Haiku               | 19.00            | 21.39                       |
+| FW0469 Haiku               | 19.00            | 21.39                       |
+| FW0470 Haiku fitted        | 19.00            | 21.39                       |
+| FW0471 Haiku fitted        | 19.00            | 21.39                       |
+| FW0472 Haiku               | 18.50            | 20.83                       |
+| FW0473 Haiku organic tight | 19.00            | 21.39                       |
+| FW0474 Haiku Denim         | 20.00            | 22.52                       |
+| FW0475 Haiku               | 25.50            | 28.71                       |
+| FW0476 Haiku               | 29.50            | 33.22                       |
+| FW0477 Haiku               | 36.00            | 40.54                       |
+| FW0478 Haiku H Anniversary | 19.00            | 21.39                       |
+| FW0479 Haiku 20 years leaf | 19.00            | 21.39                       |
+| FW0480 (Sticker)           | 1.00             | 1.13                        |
+| FW0481 (Sticker)           | 1.00             | 1.13                        |
+| FW0482 (Sticker)           | 1.00             | 1.13                        |
+
+The prices (in Euros only) in the image below are the same as the table above. Note that the “,” used in the prices in the image represents a decimal point, thus “18,50 €” is eighteen (18) Euros and fifty (50) cents.   
+
+<img src="/files/Haiku Project_prices.png"/>  
+
+<br>
+
+[**Anyone who is interested in purchasing the merchandise may place orders at Freewear's online store here**](https://www.freewear.org/Haiku).
+
+**ENDS**
+
+For media enquiries, please contact:   
+contact(@)haiku-os.org (The Haiku Project)   
+admin(@)freewear.org (FreeWear)  
+
+
+## NOTES TO EDITORS
```

Comment:
This and all things below are fine for the press release, but should probably not be copied on our own website?